src/main.rs

```diff
@@ -361,22 +366,30 @@ fn handle_window_event(window: &sdl2::video::Window,
         Event::KeyDown{keycode: Some(Keycode::Backquote), ..} => {
             game.console.lock().unwrap().toggle();
         }
-        Event::KeyDown{keycode: Some(key), ..} => {
+        Event::KeyDown{keycode: Some(key), keymod, ..} => {
             if game.focused {
                 if let Some(steven_key) = settings::Stevenkey::get_by_keycode(key, &game.vars) {
                     game.server.key_press(true, steven_key);
                 }
             } else {
-                ui_container.key_press(game, key, true);
+                let mut ctrl_pressed = false;
+                if keymod.intersects(keyboard::LCTRLMOD | keyboard::RCTRLMOD) {
+                    ctrl_pressed = true;
+                }
```

Comment:
this could be `let ctrl_pressed = keymod.intersects(keyboard::LCTRLMOD | keyboard::RCTRLMOD)` (and below too)